addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1112,8 +1512,14 @@ var SnippetEditor = Widget.extend({
         this.$dropZones.droppable({
             over: function () {
                 if (self.dropped) {
+                    // Checking if the "out" event happened before this "over":
+                    // if `self.dropped` is "true", "out" didn't happen because
+                    // it sets it to "false". We are therefore in the case of an
+                    // "over" after an "over" and we need to escape the previous
+                    // dropzone first.
                     self.$target.detach();
                     $('.oe_drop_zone').removeClass('invisible');
```

Comment:
Since [this commit](https://github.com/odoo/odoo/commit/31cc10b), the website is inside of an iframe. Due
to that, the snippets menu and snippet editor do not have a direct access to the dropzones. This has been fixed in [this PR](https://github.com/odoo/odoo/pull/136644)